package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "build": "set -ex; tsc --build",
+    "build": "set -ex; yarn workspace @prodo/devtools-core build; tsc --build",
```

Comment:
How is your process of re-transpiling the devtools when files changes? Is there a similar `--watch` for babel?

Reply:
Uhh, it may be manual at the moment. Will look into it :)